bootstrap.sh

```diff
@@ -26,6 +26,34 @@ dotnet restore
 
 ./build.sh -p -f
 
+info=$(dotnet --info)
+
+
+
+if echo $info | grep -q -i 'osx';
+then
+    # download osx version of clang-tidy/format
+    wget https://clrjit.blob.core.windows.net/clang-tools/osx/clang-format.exe -O bin/clang-format.exe
+    wget https://clrjit.blob.core.windows.net/clang-tools/osx/clang-tidy.exe -O bin/clang-tidy.exe
+elif echo $info | grep -q -i 'ubuntu.16.04';
+then
+    # download osx version of clang-tidy/format
+    wget https://clrjit.blob.core.windows.net/clang-tools/ubuntu/16.04/clang-format.exe -O bin/clang-format.exe
+    wget https://clrjit.blob.core.windows.net/clang-tools/ubuntu/16.04/clang-tidy.exe -O bin/clang-tidy.exe
+elif echo $info | grep -q -i 'ubuntu';
+then
+    # download osx version of clang-tidy/format
+    wget https://clrjit.blob.core.windows.net/clang-tools/ubuntu/14.04/clang-format.exe -O bin/clang-format.exe
+    wget https://clrjit.blob.core.windows.net/clang-tools/ubuntu/14.04/clang-tidy.exe -O bin/clang-tidy.exe
+elif echo $info | grep -q -i 'centos';
+then
+    # download osx version of clang-tidy/format
+    wget https://clrjit.blob.core.windows.net/clang-tools/centos/clang-format.exe -O bin/clang-format.exe
+    wget https://clrjit.blob.core.windows.net/clang-tools/centos/clang-tidy.exe -O bin/clang-tidy.exe
+else
+    echo "Clang-tidy and clang-format not installed. Please install and put them on the PATH to use jit-format."
+fi
+
```

Comment:
So it's not just the windows bootstrap that downloads the tools.   Should there be some kind of check to make sure that they're not already on the path?